streams/test-utils/src/test/java/org/apache/kafka/streams/internals/KeyValueStoreFacadeTest.java

```diff
@@ -53,9 +54,9 @@ public void setup() {
         keyValueStoreFacade = new KeyValueStoreFacade<>(mockedKeyValueTimestampStore);
     }
 
-    @SuppressWarnings("deprecation") // TODO will be fixed in KAFKA-10437
+    @SuppressWarnings("deprecation") // test of deprecated method
```

Comment:
My prior comment was wrong. We do want to test the deprecated init method as well as the non-deprecated one.